ci/cirrus.sh

```diff
@@ -31,13 +31,10 @@ print_environment() {
 }
 print_environment
 
-# Start persistent wineserver if necessary.
-# This speeds up jobs with many invocations of wine (e.g., ./configure with MSVC) tremendously.
 case "$WRAPPER_CMD" in
     *wine*)
         # Make sure to shutdown wineserver whenever we exit.
         trap "wineserver -k || true" EXIT INT HUP
-        wineserver -p
         ;;
 esac
```

Comment:
Oh, I didn't mean to remove this code. Starting the wineserver is still useful, because it speeds up the tests.  I just meant that we can drop these lines:
```
        # Wait until the wineserver process has
        # exited, to avoid corrupting the wine prefix.
        while (ps -A | grep wineserver) > /dev/null; do sleep 1; done
```

Reply:
Sorry for constantly offering tiny suggestions that break stuff... -.- 

What happens here is that `wineserver -p` fails because there's a server running already (from the `wineboot` still, but it's not a persistent one...) While the "wait until the wineserver process has exited" code was still there, this didn't happen.

I took a step back and did some testing on my local machine:
 - `wineserver -p` is at most useful with MSVC builds. With mingw builds, there won't be many wine invocations. And even with MSVC builds, it's questionable: The server stays around for 3 seconds and waits for another wine process to appear before it exits, so this should be necessary only when things are really slow. This has indeed speed up builds for us in the past, but maybe this was because some other part didn't work... No idea. 
 - `wineboot --init` doesn't seem to be required. The first wine invocation should init stuff automatically (and this is how I know wine...)

So I suggest to just remove all of this and pushed it as a fixup here: https://github.com/real-or-random/secp256k1/tree/230815-gha-msvc . CI still in my repo running, but I hope that it just works now...

Reply:
https://github.com/real-or-random/secp256k1/commit/76ecd47c42e320cab53ae30dce18710ad3a45ff1 has been cherry-picked and squashed with the last commit.